src/components/proofEditor/issueInformation.tsx

```diff
@@ -7,14 +8,18 @@ export interface Props {
 	issue: Issue;
 }
 
-export class IssueInformation extends React.Component<Props> {
+export class IssueInformation extends React.Component<Props, {}> {
 
 	public render() {
```

Comment:
I would wish for at least one or two lines of comment explaining what's happens exactly in this rendermethod

Reply:
Now that you understand the concepts used here do you think it is still necessary?